test/library/tomography/test_process_tomography.py

```diff
@@ -361,6 +353,12 @@ def test_analysis_config(self):
         self.assertNotEqual(analysis, loaded)
         self.assertEqual(analysis.config(), loaded.config())
 
+    def test_exp_serialization(self):
```

Comment:
I would leave such test for another PR and focus on circuits only in this one

Reply:
I deleted the test as suggested.